### ISSUE TITLE
Are you still supporting this project?

### ISSUE THREAD

**Issue by rimutaka** [colbyn/dynamodb_data#1]
Hi @colbyn. Thank you for putting together this neat solution. I would prefer it over a more popular _serde_dynamodb_. What is the status of this project? Do you still use it yourself? Would you recommend it?

**Comment by colbyn** [comment]
No, probably not. I wrote this when working on a SAAS project. But not anymore. (Unless I suppose I’m getting paid to do this or happen to use this again at a job.) Though considering the simplicity of the API, the implementation is very simple for anyone who wants to fork this crate.
